geoh5py/objects/vp_model.py

```diff
@@ -165,6 +165,28 @@ def _promote_uuid_attribute(
 
         return data.uid
 
+    @classmethod
+    def create(cls, workspace, **kwargs):
+        """
+        Function to create an entity.
+
+        The visual parameters are set to default values, and the filter_basement
+        is set to 5% of the vertical extent of the model.
+
+        :param workspace: Workspace to be added to.
+        :param kwargs: List of keyword arguments defining the properties of a class.
+
+        :return entity: Registered Entity to the workspace.
+        """
+        new_object = super().create(workspace, **kwargs)
+
+        viz_params = new_object.add_default_visual_parameters()
+        viz_params.filter_basement = (
```

Comment:
can you just explain the logic behind this defualt value?

Reply:
See the docstrings of the method above. We need to assign some "small" value, which I decided to define as 5% of the total Vpmodel depth.